lib/include/config.h

```diff
@@ -24,7 +24,8 @@
 #elif defined(MAC_OS)
     #define COMMONS_CRYPTO_OPENSSL_LIBRARY "libcrypto.dylib"
 #else
-    #define COMMONS_CRYPTO_OPENSSL_LIBRARY "libcrypto.so"
+    #define COMMONS_CRYPTO_OPENSSL_LIBRARY "libcrypto.so.1.0.0"
+    #define COMMONS_CRYPTO_OPENSSL_LIBRARY_1 "libcrypto.so.1.1"
```

Comment:
I think I would be happier with COMMONS_CRYPTO_OPENSSL_LIBRARY_1_0 and   COMMONS_CRYPTO_OPENSSL_LIBRARY_1_0

thoughts?